src/bin/pg_autoctl/cli_do_tmux.c

```diff
@@ -245,6 +270,68 @@ tmux_add_send_keys_command(PQExpBuffer script, const char *fmt, ...)
 }
 
 
+/*
+ * tmux_prepare_XDG_environment set XDG environment variables in the current
+ * process tree.
+ */
+static bool
+tmux_prepare_XDG_environment(const char *root)
```

Comment:
Something weird is going on with this. If I run two clusters at the same time, the second cluster will use the XDG variables from the first somehow. To reproduce:
```bash
#shell 1
make -j12 cluster
# shell 2
make TMUX_TOP_DIR=tmux2 FIRST_PGPORT=6500 cluster -j9
```

The second cluster will then use the XDG paths of the first. I guess the reason for this is that the tmux daemon is already running and thus using `setenv` doesn't have any effect for the second cluster. So I think it's probably better to set them always using `sendkeys` again.

Reply:
I think I was totally confused with the tmux process model. If the tmux server is started at the right time then it all makes sense with the previous coding, but when the tmux server has already been running then it keeps the first XDG values. I had strange test cases locally, and I think that's what happened. Should be fixed now.